iceoryx_examples/waitset_in_c/ice_c_waitset_individual.c

```diff
@@ -61,16 +61,19 @@ int main()
     iox_sub_t subscriber[NUMBER_OF_SUBSCRIBERS];
 
     // create two subscribers, subscribe to the service and attach them to the waitset
-    const uint64_t historyRequest = 1U;
-    const uint64_t queueCapacity = 256U;
-
-    const char* const nodeName1 = "iox-c-ex-waitset-individual-node1";
-    const char* const nodeName2 = "iox-c-ex-waitset-individual-node2";
+    iox_subscriber_options_t options1, options2;
+    options1.historyRequest = 1U;
+    options1.queueCapacity = 256U;
+    options1.nodeName = "iox-c-ex-waitset-individual-node1";
+    
+    options2.historyRequest = 1U;
+    options2.queueCapacity = 256U;
+    options2.nodeName = "iox-c-ex-waitset-individual-node2";
```

Comment:
could `option1` be reused by just setting `nodeName` after `iox_sub_init`

Reply:
Yes but I did not want to confuse the user.